elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -106,4 +109,8 @@ public function filter__jetpack_active_modules( $modules ) {
 		// array_filter() preserves keys, so to get a clean / flat array we must pass it through array_values()
 		return array_values( $filtered );
 	}
+
+	public function filter__jetpack_widgets_to_include( $widget_include ) {
+		return [];
```

Comment:
This is going to disable _every_ JP widget - we only need to disable the Search one.

Reply:
Switched over to only disabling search in 2bd452fc